docs/data/material/components/material-icons/SearchIcons.js

```diff
@@ -110,6 +111,12 @@ const StyledIcon = styled('span')(({ theme }) => ({
   },
 }));
 
+const PlaceholderIcon = styled('div')({
+  width: 69,
+  height: 69,
```

Comment:
This could be better, avoiding magic numbers a bit more.

```suggestion
  width: iconWidth + 8 * 2 * 2,
  height: iconWidth + 8 * 2 * 2,
```

Reply:
Replacing one magic number with five? 😄